pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: mockito
-version: 3.0.0-alpha
+version: 3.0.0
```

Comment:
Why change this?

Reply:
alpha has already been released.

Reply:
True. Let's call it `3.0.0-dev` for now, and not release it. 3.0.0 will be released not super soon, like March probably. I don't want anyone with an itchy trigger finger to publish 3.0.0 till it's ready.

Reply:
Done.